### PR TITLE
Feature/eq views

### PR DESCRIPTION
**Pull request by elmiquea**
added material to main/dashboard-handlebars and css

### CODE REVIEW

File: views/layouts/dashboard.handlebars [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  <header class="hero">
+  </header>
+  <nav class="navbar">
+    <ul>
+      <li class="li-inline nav-item"><h1 class="app-title"><a href="/">NODOMON-CHOICE-RPG</a></h1></li>
+      <li class="li-inline nav-item"><a href="/">Home</a></li>
+      <li class="li-inline nav-item"><a href="/Sign up">sign up</a></li>
+      <li class="li-inline nav-item"><a href="/login">Login</a></li>
+      <li class="li-inline nav-item"><a href="/adventure">adventure</a></li>
+      <li class="li-inline nav-item"><a href="/change character">change character</a></li>
```
Comment (Blossomswilts) [inline views/layouts/dashboard.handlebars]: Here should just be a character, not change character

File: views/layouts/dashboard.handlebars [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>THE NODOMON-CHOICE</title>
```
Comment (Blossomswilts) [inline views/layouts/dashboard.handlebars]: Please change this from THE NODOMON-CHOICE to just Nodomon

Same with the links.

File: views/layouts/dashboard.handlebars [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+</head>
+
+<body class="flex-column min-100-vh">
+  <header class="hero">
+  </header>
+  <nav class="navbar">
+    <ul>
+      <li class="li-inline nav-item"><h1 class="app-title"><a href="/">NODOMON-CHOICE-RPG</a></h1></li>
+      <li class="li-inline nav-item"><a href="/">Home</a></li>
+      <li class="li-inline nav-item"><a href="/Sign up">sign up</a></li>
```
Comment (Blossomswilts) [inline views/layouts/dashboard.handlebars]: Sign up/Login should only be one page, and not two.

File: views/layouts/main.handlebars [hunk 4]
```diff
@@ -2,14 +2,38 @@
 <html>
 
 <head>
-  <meta charset="utf-8">
-  <title>{{title}}</title>
+ <head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>NODOMON-CHOICE-RPG</title>
```
Comment (Blossomswilts) [inline views/layouts/main.handlebars]: Title change

File: views/layouts/main.handlebars [hunk 5]
```diff
@@ -2,14 +2,38 @@
...
 </head>
 
-<body>
-  {{{body}}}
-  <script src="/js/main.js"></script>
+<body class="flex-column min-100-vh">
+  <header class="hero">
+  </header>
+  <nav class="navbar">
+    <ul>
```
Comment (Blossomswilts) [inline views/layouts/main.handlebars:17]: Same changes as previous statement(s)

**Review by Blossomswilts — DISMISSED** [review]
I left some line changes within the file, please review them and make the necessary changes. 
Thank you.

**Review by elmiquea — COMMENTED** [review]
made changes to main.handlebars, dashboard and server

**Review by elmiquea — COMMENTED** [review]
changes made

**Comment by elmiquea** [comment]
made necessary changes

File: views/layouts/dashboard.handlebars [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  <link href="https://fonts.googleapis.com/css?family=IBM+Plex+Sans:400,400i,700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="/css/style.css">
+</head>
+
+<body class="flex-column min-100-vh">
+  <header class="hero">
+  </header>
+  <nav class="navbar">
+    <ul>
+      <li class="li-inline nav-item"><h1 class="app-title"><a href="/">NODOMON-CHOICE-RPG</a></h1></li>
```
Comment (Blossomswilts) [inline views/layouts/dashboard.handlebars]: Title change still needed here.

File: views/layouts/dashboard.handlebars [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+      <li class="li-inline nav-item"><a href="/">Home</a></li>
+      <li class="li-inline nav-item"><a href="/Sign up">sign up</a></li>
```
Comment (Blossomswilts) [inline views/layouts/dashboard.handlebars]: Add Log-in/Sign-up

File: views/layouts/dashboard.handlebars [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+      <li class="li-inline nav-item"><a href="/login">Login</a></li>
+      <li class="li-inline nav-item"><a href="/adventure">adventure</a></li>
```
Comment (Blossomswilts) [inline views/layouts/dashboard.handlebars]: Capitalize adventure to Adventure

File: views/layouts/dashboard.handlebars [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+      <li class="li-inline nav-item"><a href="/change character">change character</a></li>
```
Comment (Blossomswilts) [inline views/layouts/dashboard.handlebars]: Remove change and capitalize character to Characters

I'm thinkiing we keep this plural

File: views/layouts/main.handlebars [hunk 10]
```diff
@@ -2,14 +2,37 @@
...
+      <li class="li-inline nav-item"><h1 class="app-title"><a href="/">NODOMON-CHOICE-RPG</a></h1></li>
+      <li class="li-inline nav-item"><a href="/">Home</a></li>
+      <li class="li-inline nav-item"><a href="/login">Login</a></li>
+      <li class="li-inline nav-item"><a href="/adventure">adventure</a></li>
+      <li class="li-inline nav-item"><a href="/change character">change character</a></li>
```
Comment (Blossomswilts) [inline views/layouts/main.handlebars]: to Character

File: views/layouts/main.handlebars [hunk 11]
```diff
@@ -2,14 +2,37 @@
...
-  <script src="/js/main.js"></script>
+<body class="flex-column min-100-vh">
+  <header class="hero">
+  </header>
+  <nav class="navbar">
+    <ul>
+      <li class="li-inline nav-item"><h1 class="app-title"><a href="/">NODOMON-CHOICE-RPG</a></h1></li>
+      <li class="li-inline nav-item"><a href="/">Home</a></li>
+      <li class="li-inline nav-item"><a href="/login">Login</a></li>
+      <li class="li-inline nav-item"><a href="/adventure">adventure</a></li>
```
Comment (Blossomswilts) [inline views/layouts/main.handlebars]: Capitalize to Adventure

File: views/layouts/main.handlebars [hunk 12]
```diff
@@ -2,14 +2,37 @@
...
 
-<body>
-  {{{body}}}
-  <script src="/js/main.js"></script>
+<body class="flex-column min-100-vh">
+  <header class="hero">
+  </header>
+  <nav class="navbar">
+    <ul>
+      <li class="li-inline nav-item"><h1 class="app-title"><a href="/">NODOMON-CHOICE-RPG</a></h1></li>
```
Comment (Blossomswilts) [inline views/layouts/main.handlebars]: Change to Donomon

File: views/layouts/main.handlebars [hunk 13]
```diff
@@ -2,14 +2,37 @@
...
+      <li class="li-inline nav-item"><a href="#" id="logout-link">Logout</a></li>
+    </ul>
+  </nav>
+
+  <main class="col-auto">
+    {{{body}}}
+  </main>
+
+  <footer class="flex-row justify-center">
+    &copy; Monodomon-Choice-RPG
```
Comment (Blossomswilts) [inline views/layouts/main.handlebars]: Change to Donomon

File: views/layouts/main.handlebars [hunk 14]
```diff
@@ -2,14 +2,37 @@
 <html>
 
 <head>
-  <meta charset="utf-8">
-  <title>{{title}}</title>
+ <head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>NODOMON</title>
```
Comment (Blossomswilts) [inline views/layouts/main.handlebars]: Change to Donomon
I mistakenly said nodomon before

File: views/layouts/dashboard.handlebars (hunk 6 above)
Comment (Blossomswilts) [reply views/layouts/dashboard.handlebars]: Donomon **

**Review by Blossomswilts — CHANGES_REQUESTED** [review]
I added some more minor changes, such as Nodomon to Donomon. That was my bad.
Also just fix punctuation such as capitilization where needed, and combine signup and longin to one page instead of two on every page. 
Thank you.

**Review by elmiquea — COMMENTED** [review]
Name changed and spelling.

File: views/layouts/dashboard.handlebars (hunk 8 above)
Comment (elmiquea) [reply views/layouts/dashboard.handlebars]: done

**Review by elmiquea — COMMENTED** [review]
changes made

File: views/layouts/main.handlebars (hunk 14 above)
Comment (elmiquea) [reply views/layouts/main.handlebars]: done

**Review by elmiquea — COMMENTED** [review]
done

**Review by Blossomswilts — APPROVED** [review]

**Pull request merged**